### PR TITLE
[WIP] Add ground station communication packages

### PR DESCRIPTION
**Pull request by Sedictious**

### CODE REVIEW

File: src/poliastro/czml/utils.py [hunk 1]
```diff
@@ -30,3 +30,89 @@ def ellipsoidal_to_cartesian(a, b, lat, lon):
...
+
+    Returns
+    -------
+    This returns both of the points intersecting the ellipsoid.
+
+    """
+    # Get rid of one parameter by translating the line's direction vector
+    k, l = u2 / u1, u3 / u1
+    t0 = (
+        -np.power(a, 2) * np.power(b, 2) * l * z
```
[the remark this answers is not in the document]
Comment (Sedictious) [reply src/poliastro/czml/utils.py]: Will do! I initially thought that ``np.power`` is more accurate but turns out it's not and inlining is much faster anyway 😉

File: src/poliastro/czml/utils.py [hunk 2]
```diff
@@ -30,3 +30,89 @@ def ellipsoidal_to_cartesian(a, b, lat, lon):
     y = cr * np.cos(lat) * np.sin(lon)
     z = (a / b) ** 2 * cr * np.sin(lat)
     return x, y, z
+
+
+def intersection_ellipsoid_line(a, b, c, u1, u2, u3, x, y, z):
```
[the remark this answers is not in the document]
Comment (Sedictious) [reply src/poliastro/czml/utils.py]: It's just the solution of the system of the ellipsoid and the line. Do you think there's still a need to find a source to cite?

File: src/poliastro/czml/czml_extract_default_params.py [hunk 3]
```diff
@@ -75,3 +75,25 @@
         "map_url": "",
     },
 }
+
+ACCESS_PACKET = {
+    "id": "FKC7SQFxL5",
```
[the remark this answers is not in the document]
Comment (Sedictious) [reply src/poliastro/czml/czml_extract_default_params.py]: Yep, just a simple hash. The packet is not really necessary, but it allows us to easily access all the entitie's children from the application.

File: src/poliastro/czml/utils.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Juanlu001) [reply src/poliastro/czml/utils.py]: It's not crucial, but given that the formula is quite long, it would be nice to have it :)

**Pull request closed without merging**